### PR TITLE
Run `gradle init` to generate build.gradle from pom.xml

### PR DESCRIPTION
**Pull request by manics**
Create `build.gradle` following https://medium.com/@jayanisumudini/convert-maven-project-to-gradle-project-d1a829d7637f

### CODE REVIEW

**Comment by dominikl** [comment]
Seems to work fine, thanks @manics 👍 Would be good to get this merged.

**Comment by mtbc** [comment]
Could update README.md with how to build and run?

File: build.gradle [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        url = 'http://artifacts.openmicroscopy.org/artifactory/maven'
+    }
+    maven {
+        url = 'http://repo.maven.apache.org/maven2'
+    }
+}
+
+dependencies {
+    implementation 'org.openmicroscopy:omero-server:5.5.6'
+    implementation 'com.google.guava:guava:27.1-jre'
```
Comment (jburel) [inline build.gradle:18]: is that needed?

Comment (jburel) [reply build.gradle:18]: also in other repo we have used implementation()

File: build.gradle [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    implementation 'io.vertx:vertx-core:3.9.0'
+    implementation 'io.vertx:vertx-web:3.9.0'
+}
+
+group = 'org.openmicroscopy'
+version = '0.1.0-SNAPSHOT'
+description = 'omero-ms-zarr'
+sourceCompatibility = '1.8'
```
Comment (jburel) [inline build.gradle:26]: java {
    sourceCompatibility = JavaVersion.VERSION_1_8
}

File: build.gradle (hunk 1 above)
Comment (mtbc) [reply build.gradle:18]: Seems likely, can revisit once more code's written. Or just remove in a separate commit then I can revert when I use it.

**Comment by dominikl** [comment]
Sorry, have to correct myself, it doesn't work... used the previously jar generated from maven.
Using the gradle one I get: `no main manifest attribute, in build/libs/omero-ms-zarr-0.1.0-SNAPSHOT-all.jar`

File: build.gradle (hunk 2 above)
Comment (manics) [reply build.gradle:26]: What does this do?

File: build.gradle (hunk 1 above)
Comment (manics) [reply build.gradle:19]: I've left this in since it was in the pom.xml that this was created from. We can update it in a later PR if necessary.

**Comment by manics** [comment]
@dominikl It should run now, see the updated README. Note the suggested `/etc/omero.properties` argument is just copied from the existing README, if it doesn't work it'll be best if someone else adds the correct invocation.

File: build.gradle (hunk 2 above)
Comment (jburel) [reply build.gradle:27]: it indicates that it is for the java plugin

Comment (manics) [reply build.gradle]: Done

**Comment by dominikl** [comment]
👍 Indeed, works now.

File: build.gradle [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    id 'application'
+    id 'maven-publish'
+    id 'com.github.johnrengelman.shadow' version '5.2.0'
+}
+
+repositories {
+    maven {
+        url = 'http://artifacts.openmicroscopy.org/artifactory/maven'
+    }
+    maven {
```
Comment (jburel) [inline build.gradle:12]: could use ``mavenCentral()`` instead?

Comment (manics) [reply build.gradle]: Done

**Comment by joshmoore** [comment]
Green. Merging.

**Pull request merged**